docs/evaluation/how_to_guides/multiturn_evaluation.mdx

```diff
@@ -0,0 +1,51 @@
+# How to set up a multiturn evaluation
+
+LangSmith makes it easy to evaluate multiturn conversations in the playground. This allows you to evaluate how changing your system prompt,
+the tools available to the model, or the output schema affects a conversation with multiple messages.
+
+This how-to guide walks you through the various ways you can set up the playground for multiturn evaluation, depending on where your
+conversation data is stored.
+
```

Comment:
I think it would be nice to have a screenshot here of what a multi-turn conversation looks like